language/de/acp/posting.php

```diff
@@ -45,7 +45,7 @@
 	'ACP_BBCODES_EXPLAIN'		=> 'BBCode ist eine spezielle HTML-Implementierung, die eine größere Kontrolle über das, was angezeigt wird, bietet. Hier kannst du benutzerdefinierte BBCodes hinzufügen, ändern oder entfernen.',
 	'ADD_BBCODE'				=> 'BBCode hinzufügen',
 
-	'BBCODE_DANGER'				=> 'Der BBCode, den du anlegen möchtest, scheint ein {TEXT}-Token innerhalb eines HTML-Attributs zu nutzen. Dies ist ein möglicher Angriffspunkt für Cross-Site Scripting (XSS). Verwende stattdessen, wenn möglich, die restriktiveren {SIMPLETEXT}- oder {INTTEXT}-Typen. Fahre nur fort, wenn du dich dem Risiko bewusst bist und die Verwendung von {TEXT} unvermeidbar ist.',
+	'BBCODE_DANGER'				=> 'Der BBCode, den du anlegen möchtest, scheint unsicher zu sein. Wenn der BBCode ein {TEXT}-Token in einer heiklen Position verwendet, versuche stattdessen einen mehr restriktiven Token zu verwenden. Fahre nur fort, wenn du dir der Risiken bewusst bist.',
```

Comment:
"heikle Position" klingt irgendwie ungünstig. Hat jemand eine gute Idee?

Reply:
Der BBCode, den du anlegen möchtest, scheint nicht sicher zu sein. Wenn der BBCode ein {TEXT}-Token in einer sensiblen Position verwendet, versuche stattdessen, ein restriktiveres Token zu verwenden. Fahre nur fort, wenn du dir der Risiken bewusst bist.

Reply:
Hm, wieso ist die bisherige Version denn geändert worden? Ich finde den alten Text sehr viel klarer, was das detektierte Problem ist. Ansonsten finde ich phillips Version aber gut.